lib/features/support/env.rb

```diff
@@ -32,7 +33,10 @@
 After do |scenario|
   kill_script
 
-  write_failed_requests_to_disk(scenario) if scenario.failed?
+  if scenario.failed?
+    write_failed_requests_to_disk(scenario)
+    output_logs if defined? output_logs
```

Comment:
This is a very good idea :) I think this will be really handy for getting information on what the state of the system is like when the iOS simulator does weird things.